client/my-sites/plan-testimonials/index.jsx

```diff
@@ -18,7 +18,7 @@ export class PlanTestimonials extends Component {
 				</div>
 				<div className="plan-testimonials__content-right-container">
 					<div className="plan-testimonials__content-right">
-						Did you know that WordPress powers 36% of the entire internet? WordPress.com is the best
+						Did you know that WordPress powers 35% of the entire internet? WordPress.com is the best
```

Comment:
It will be good to have that number as a constant somewhere like we have on the server.

Reply:
We recently wrapped up an AB test in https://github.com/Automattic/wp-calypso/pull/41224 that makes this file obsolete. AFAIK we don't use the "WordPress powers..." text anywhere else in Calypso right now.